build/build.go

```diff
@@ -696,7 +725,7 @@ func (db *DeploymentBuilder) filterHost(profile *starlingxv1.HostProfile, host *
 	return nil
 }
 
-func (db *DeploymentBuilder) filterPlatformNetworks(platform_network *starlingxv1.PlatformNetwork, deployment *Deployment) error {
+func (db *DeploymentBuilder) filterPlatformNetworks(platform_network *starlingxv1.PlatformNetwork, address_pool *starlingxv1.AddressPool, deployment *Deployment) error {
```

Comment:
I think `addrpool` is unnecessary argument, please remove if it's not used.

Reply:
Removed addrpool